.contrib/Parser/DATAS/02 - Outdoor Zones/05 Pandaria/The Jade Forest/Achievements.lua

```diff
@@ -39,6 +39,7 @@ _.Zones =
 							{ 44.9, 64.6, THE_JADE_FOREST },
 							{ 47.1, 67.5, THE_JADE_FOREST },
 							{ 46.2, 71.2, THE_JADE_FOREST },
+							{ 44.5, 64.4, THE_JADE_FOREST },
```

Comment:
This is awfully close to the first coord, did you mean `45.5, 64.4`?

Reply:
Negative.
This coordinate was previously unknown within the wowhead section of this page. As the distance between this and the first entr is 44 yard, for an item that is extremely tiny and unnoticable, I feel it's validly appropriate to add it.